### PR TITLE
Add questionnaire

### PR DESCRIPTION
**Pull request by fl3pp**

### CODE REVIEW

File: SA/concept_questionnaire/concept_questionnaire.typ [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+#let question(text) = [
+  #v(1em)
+  #text
+
+  #box(width: 1fr, repeat[.])
+]
+
+#let questions = [
+  #question("Where you able to understand the meaning of the boxes and arrows?")
```
Comment (lstreckeisen) [inline SA/concept_questionnaire/concept_questionnaire.typ:144]:
```suggestion
  #question("Were you able to understand the meaning of the boxes and arrows?")
```

File: SA/concept_questionnaire/concept_questionnaire.typ [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+#[
+= VisualFP Concept Questionare
+
+Hi there,
+
+In the context of our SA, we are currently searching for a new way to
+visualize functional programming concepts. Before we start to flesh out our
+ideas, we would like to get some feedback on a few concepts we came up with.
+
```
Comment (lstreckeisen) [inline SA/concept_questionnaire/concept_questionnaire.typ:61]: We'll then decide which concept we'll develop further based on the received feedback.

File: SA/concept_questionnaire/concept_questionnaire.typ [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+In the end we will decide on one and develop it further. In the end, we will
```
Comment (lstreckeisen) [inline SA/concept_questionnaire/concept_questionnaire.typ:62]:
```suggestion
In the end, we will
```

File: SA/concept_questionnaire/concept_questionnaire.typ [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+have developed a concept, along with a proof of concept of it, that
```
Comment (lstreckeisen) [inline SA/concept_questionnaire/concept_questionnaire.typ:63]:
```suggestion
have developed a design concept, along with a proof of concept of it, that
```

File: SA/concept_questionnaire/concept_questionnaire.typ [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+fullfills the following criteria:
+
+1. It can be used to teach functional programming concepts
```
Comment (lstreckeisen) [inline SA/concept_questionnaire/concept_questionnaire.typ:66]: I would drop the criteria. I'm really not sure if the PoC can already be used for teaching.

File: SA/concept_questionnaire/concept_questionnaire.typ [hunk 6]
```diff
@@ -0,0 +1,157 @@
...
+    product (n : ns) = (*) n (product ns)
+    ```
+  ],
+  map_add_5_function: [
+    ```haskell
+    mapAdd5 :: Num a => [a] -> [a]
+    mapAdd5 = map (+ 5)
+    ```
+  ],
+  count_words: [
```
Comment (lstreckeisen) [inline SA/concept_questionnaire/concept_questionnaire.typ:112]: Maybe we should drop this one. It doesn't include any concepts the other scenarios don't have (it's just larger) and we don't have a visualization for it (or at least I don't).

File: SA/concept_questionnaire/concept_questionnaire.typ (hunk 2 above)
Comment (fl3pp) [reply SA/concept_questionnaire/concept_questionnaire.typ:70]: added

File: SA/concept_questionnaire/concept_questionnaire.typ (hunk 5 above)
Comment (fl3pp) [reply SA/concept_questionnaire/concept_questionnaire.typ:74]: reworded

File: SA/concept_questionnaire/concept_questionnaire.typ (hunk 6 above)
Comment (fl3pp) [reply SA/concept_questionnaire/concept_questionnaire.typ:120]: agree, then we'll have room for the questions on the same page

**Comment by fl3pp** [comment]
Important! Squash this PR, because the font would be checked in outside of LFS otherwise

**Review by lstreckeisen — APPROVED** [review]

**Pull request merged**